### ISSUE TITLE
Difference to npm's overrides unclear

### ISSUE THREAD

**Issue by stefanpl** [rogeriochaves/npm-force-resolutions#50]
Coming to this package from a stackoverflow post.

For me, it's hard to tell why this package exists. It seems redundant to what [overrides](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides) do in npm natively.

I'm wondering:
- was this a legacy solution that existed before overrides?
- can this do anything that overrides can't?
- did I misunderstand the whole point of either one of them?

Maybe (bold assumption: probably) I'm not the only one having these questions. I'd appreciate it very much to have a short notice mentioning the relation to overrides somewhere in the readme.

🙏

**Comment by elcreator** [comment]
There are many environments where npm version lesser than 8 should be used.

**Comment by runderworld** [comment]
@stefanpl 

> was this a legacy solution that existed before overrides?

precisely.  this is the entire reason this library was built, to provide the functionality before it was publicly known that NPM maintainers would introduce the feature themselves.

> can this do anything that overrides can't?

no, it's more limited than built-in npm`overrides`.
